src/generic/components/ecl-navigation/ecl-navigation-side/ecl-navigation-side.scss

```diff
@@ -0,0 +1,253 @@
+    display: none;
+
+    // Force display if no JS
+    .no-js & {
+      display: flex;
+    }
+  }
+
+  .ecl-navigation-side__toggle {
+    border-width: 0;
+    color: #fff;
+    font-size: map-get($ecl-font-size, 'm');
+    font-weight: bold;
+    padding: map-get($ecl-spacing, 'm') map-get($ecl-spacing, 's');
+    position: relative;
+    text-align: left;
+    text-decoration: none;
+    width: 100%;
+    z-index: 1;
+  }
+
+  .ecl-navigation-side__toggle::before {
+    background-color: map-get($ecl-colors, 'blue-75');
+    content: '';
+    height: 100%;
+    left: 50%;
+    position: absolute;
+    top: 0;
+    transform: translateX(-50%);
+    width: 100vw;
+  }
+
+  .ecl-navigation-side__group {
+    display: none;
+    list-style: none;
+    margin-left: -(map-get($ecl-spacing, 's'));
+    margin-right: -(map-get($ecl-spacing, 's'));
+    padding: 0 map-get($ecl-spacing, 's');
+    position: relative;
+  }
+
+  .ecl-navigation-side__group::before {
+    background-color: map-get($ecl-colors, 'grey-5');
+    content: '';
+    height: 100%;
+    left: 50%;
+    position: absolute;
+    top: 0;
+    transform: translateX(-50%);
+    width: 100vw;
+  }
+
+  .ecl-navigation-side__link {
+    border-bottom: 1px solid map-get($ecl-colors, 'grey-50');
+    color: map-get($ecl-colors, 'grey-100');
+    display: block;
+    font-size: map-get($ecl-font-size, 'm');
+    padding: map-get($ecl-spacing, 's') 0;
+    position: relative;
+    text-decoration: none;
+
+    &:hover,
+    &:focus {
+      text-decoration: underline;
+    }
+
+    &::after {
+      position: absolute;
+      right: 0;
+      text-decoration: none;
+    }
+
+    &[aria-expanded='true'] {
+      border-bottom-width: 0;
+    }
+
+    &--active {
+      color: map-get($ecl-colors, 'blue-100');
+    }
+  }
+
+  .ecl-navigation-side__group .ecl-navigation-side__item {
+    padding-left: map-get($ecl-spacing, 's');
+  }
+
+  .ecl-navigation-side--depth-2 .ecl-navigation-side__link {
+    border-bottom-color: map-get($ecl-colors, 'grey-25');
+    font-weight: normal;
+
+    &[aria-expanded='true'],
+    &[aria-expanded='false'] {
+      font-weight: bold;
+    }
+
+    &[aria-expanded='true'] {
+      border-bottom-width: 1px;
+    }
+  }
+
+  .ecl-navigation-side--depth-3 .ecl-navigation-side__link {
+    border-bottom-width: 0;
+  }
+
+  .ecl-navigation-side--depth-1 .ecl-navigation-side__item:last-child .ecl-navigation-side__link,
+  .ecl-navigation-side--depth-2 .ecl-navigation-side__item:last-child .ecl-navigation-side__link {
+    border-bottom-width: 0;
+  }
+
+  .ecl-navigation-side--depth-3 .ecl-navigation-side__item:last-child .ecl-navigation-side__link {
+    border-bottom-width: 1px;
+  }
+
+  /* Collapsible block display */
+  .ecl-navigation-side__link[aria-expanded='true'] ~ .ecl-navigation-side__group {
+    display: block;
+  }
+
+  /* Hamburger button */
+  // Based on https://github.com/jonsuh/hamburgers
+  .ecl-navigation-side__hamburger-box {
```

Comment:
`hamburger` seems to be a component that could be broken down to a smaller component.

Reply:
could be done indeed, if we keep both horizontal and side menu.
I'd suggest to do it in a follow-up ticket